### PR TITLE
Added notification badges for unread counts

### PR DESCRIPTION
**Pull request by zachatrocity**
image

### CODE REVIEW

**Review by swmarks — COMMENTED** [review]
The latest passing ci build isn't loading on my phone when installed fresh. I just get a white screen.

File: lib/pages/inbox/inbox_store.dart [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        ))
+        .then((e) => e.mentions + e.replies + e.privateMessages);
+  }
+
+  @action
+  Future<void> fetchNotifications(UserData? userData) async {
+    await LemmyApiV3(userData!.instanceHost).run(GetUnreadCount(
+      auth: userData.jwt.raw,
+    ));
+  }
```
Comment (jcgurango) [inline lib/pages/inbox/inbox_store.dart:34]: What's this function used for?

File: lib/main_common.dart [hunk 2]
```diff
@@ -29,6 +30,10 @@ Future<void> mainCommon(AppConfig appConfig) async {
     MultiProvider(
       providers: [
         MobxProvider(create: (context) => ConfigStore.load(sharedPrefs)),
+        MobxProvider(
+            create: (context) =>
+                InboxStore(accountsStore.defaultUserData!.instanceHost)
```
Comment (jcgurango) [inline lib/main_common.dart:35]: What happens here on a fresh install with no accounts?

File: lib/pages/home_page.dart [hunk 3]
```diff
@@ -27,6 +29,8 @@ class HomePage extends HookWidget {
     final theme = Theme.of(context);
     final currentTab = useState(0);
     final snackBarShowing = useState(false);
+    final accStore = useAccountsStore();
+    final inboxStore = useStore((InboxStore store) => store);
```
Comment (jcgurango) [inline lib/pages/home_page.dart:33]: Seems unnecessary to use useStore here with a selector that just selects the store. Probably no big deal but I think it'd make more sense to replicate the useAccountsStore() function in the stores.dart file with a new function called useInboxStore().

**Pull request closed without merging**